_posts/2017-12-21-14-01-netcore-beta1.md

```diff
@@ -111,15 +115,21 @@ public void Configure(IApplicationBuilder app)
 
     builder.EntitySet<Product>("Products");
 
-    app.UseMvc(routeBuilder => routeBuilder.MapODataServiceRoute("ODataRoute", null, builder.GetEdmModel()));
+    app.UseMvc(routeBuilder =>
+    {
+        routeBuilder.MapODataServiceRoute("ODataRoute", "odata", builder.GetEdmModel());
+
+        // Work-around for #1175
+        routeBuilder.EnableDependencyInjection();
+    });
 }
 {% endhighlight %}
 
 #### f. Start the OData service
 
 Start the OData service by running the project and open a browser to consume it. You should be able to get access to the service
-document at `http://host/service/` in which `http://host/service/` is the root path of your service. The metadata document
-can be accessed at `GET http://host:port/$metadata` and the products at `GET http://host:port/Products` where
+document at `http://host/odata/` in which `http://host/odata/` is the root path of your service. The metadata document
+can be accessed at `GET http://host:port/odata/$metadata` and the products at `GET http://host:port/odata/Products` where
 `host:port` is the host and port of your service, usually something like `localhost:xxxx`.
```

Comment:
`something like ` sounds list an example, so Maybe it's better to give an example port.

usually something like `localhost:12345`